docs/RC/RC.GetInteriorVehicleData/index.md

```diff
@@ -0,0 +1,161 @@
+## RC.GetInteriorVehicleData
+Type
+:	Function
+
+Sender
+:	SDL
+
+Purpose
+:	Obtain vehicle data and/or subscribe to notifications from the specified module in the specified location
+
+_**Trigger to send:**_   
+RSDL sends _RC.GetInteriorVehicleData_ to the vehicle:
+   1.	After receiving a valid _GetInteriorVehicleData_ request from remote-control application.
+   2.	Following the internal logic with the purpose to unsubscribe the application from notifications.
+
+_**Error handling:**_   
+RSDL has a logic to _read the capabilities from the predefined file_ so that to respond to the application in case _the vehicle fails to respond or responds with error_ to _RC.GetInteriorVehicleDataCapabilities_.
```

Comment:
What happens if an app does not have appHMIType defined as "REMOTE_CONTROL" and makes this RPC request?